openapi-filter.js

```diff
@@ -55,11 +55,11 @@ let argv = require('yargs')
     .version()
     .argv;
 
-if(argv.verbose) console.info('=== OpenAPI document fltering started ===','\n')
+if (argv.verbose) console.info('=== OpenAPI document fltering started ===','\n')
```

Comment:
I think `console.info` goes to STDOUT like `console.log` - we need something which (though it isn't an error) goes to STDERR. Again, the input document might not be an OpenAPI document.

Reply:
I changed it console.warn.